incubator/neo4j/templates/NOTES.txt

```diff
@@ -1,9 +1,35 @@
-You can connect to the cluster using the Cypher shell by running:
+We'll need to wait a few seconds for the Neo4j cluster to form.
+We need to see this line in all of our pods' logs:
+
+> Remote interface available at http://neo-helm-neo4j-core-2.neo-helm-neo4j.default.svc.cluster.local:7474/
+
+We can see the content of the logs by running the following command:
+
+kubectl logs -l "app=neo4j,component=core"
```

Comment:
Don't hard-code the app label:
-> `kubectl logs -l "app={{ template "neo4j.name" . }},component=core"`